web/genericcom.js

```diff
@@ -47,4 +47,10 @@ class ExternalServices extends BaseExternalServices {
   }
 }
 
-export { ExternalServices, initCom, Preferences };
+class MLManager {
+  async guess() {
+    return null;
+  }
+}
```

Comment:
You'll need to add the same code in https://github.com/mozilla/pdf.js/blob/master/web/chromecom.js as well.